web/src/components/health/health-color-square.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { valueFormat } from '../../utils/format';
+import { AlertWithRuleName, computeAlertScore, computeExcessRatioStatusWeighted } from './health-helper';
+
+import { Tooltip } from '@patternfly/react-core';
+import './health-color-square.css';
+
+export interface HealthColorSquareProps {
+  alert: AlertWithRuleName;
+}
+
+// rgb in [0,255] bounds
+type Color = { r: number; g: number; b: number };
+type ColorMap = Color[];
+
+const criticalColorMap: ColorMap = [
+  { r: 250, g: 234, b: 232 },
+  { r: 163, g: 0, b: 0 },
+  { r: 44, g: 0, b: 0 },
+  { r: 20, g: 0, b: 20 }
+];
+
+const warningColorMap: ColorMap = [
+  { r: 253, g: 247, b: 231 },
+  { r: 240, g: 171, b: 0 },
+  { r: 236, g: 122, b: 8 },
+  { r: 59, g: 31, b: 0 }
+];
+
+const infoColorMap: ColorMap = [
+  { r: 62, g: 134, b: 53 },
+  { r: 228, g: 245, b: 188 },
+  { r: 154, g: 216, b: 216 }
+];
```

Comment:
Wouldn't it be better to use hex colors here ?

https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient

Reply:
ok let's keep that as is here :wink: